templates/participants.html

```diff
@@ -0,0 +1,13 @@
+{% extends "right-aside.html" %}
+{% block subhead_content%}
+{% endblock subhead_content%}
+
+{% block main_content %}
+{{ page.content | safe }}
+
+{% include "includes/participants-buttons.html" %}
+<a href="https://learn.percona.com/percona-support-for-valkey" class="logo">
```

Comment:
Overall looks good. I went and added the data driven bits as a PR to your fork. https://github.com/martinrvisser/valkey-io.github.io/pull/1 , LMK what you think